satellite-placement/README.rst

```diff
@@ -0,0 +1,43 @@
+Satellite Placement Demo
+========================
+A demo on how to group a fleet of satellites into constellations of size x,
+such that the average coverage of each constellation is maximized.
+
+An alternate interpretation of this code could be: how do you group your class
+of students into teams of size x, such that you maximize the average
+performance of each team?
+
+Usage
+-----
+To run the demo,
+::
+  python satellite.py
+
+It will print out a set of satellite constellations.
+
+Code Overview
+-------------
+The idea is to consider all possible combinations of satellites, eliminate
+particularly "bad" constellations, and enforce rules on the final solution.
+These rules (or constraints) will:
+
+* Favour constellations with better coverage
+* Encourage satellites to only join *one* constellation
+* Only allow a limited number of constellations to be chosen
```

Comment:
This rule isn't clear. Is this the same as the number of solutions?
Also, and really nitpicking here, it would be nice to keep the parallelism of the previous bullets with "Allow only".